recipes/lmdb/all/conanfile.py

```diff
@@ -0,0 +1,68 @@
+import os
+import shutil
+from conans import ConanFile, CMake, tools
+
+
+class lmdbConan(ConanFile):
+    name = "lmdb"
+    license = "OLDAP-2.8"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://symas.com/lmdb/"
+    description = "Fast and compat memory-mapped key-value database"
+    topics = ("LMDB", "database", "key-value", "memory-mapped")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        root = "openldap-LMDB_{}".format(self.version)
+        os.rename(os.path.join(root, "libraries", "liblmdb"), self._source_subfolder)
+        shutil.rmtree(root)
```

Comment:
I bet theres a conan tool for this

Reply:
Yep, you are right. There is `tools.rmdir` that will do the same job. Fixed.